unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/DebugController/DebugBridge.cs

```diff
@@ -35,6 +35,8 @@ public void Setup(IDebugController debugController)
 
         public void ShowFPSPanel() { debugController.ShowFPSPanel(); }
 
+        public void SetLocalDCLVersion(string version) { debugController.SetLocalDCLVersion(version); }
```

Comment:
sure this message is needed? kernel and unity versions are already stored in `KernelConfig` you could get that data here https://github.com/decentraland/unity-renderer/blob/5a82ec43ab61d6338ea80c6e9e92da746dc5016c/unity-renderer/Assets/Scripts/MainScripts/Debugging/FPSDisplay/FPSDisplay.cs#L68